app/assets/javascripts/partials/template_edit.html.erb

```diff
@@ -13,7 +13,7 @@
 
 <div class="row no-print" style="margin-top: 2em;" ng-show="loaded">
   <div class="col-md-10 col-md-offset-1">
-    <input id="checklist-category" placeholder="Add a Category" ng-model="template.template_category.name" type="text" class="form-control" typeahead-editable="false" typeahead="category.name for category in categories | filter:$viewValue | limitTo: 5" typeahead-on-select="saveCategory($item)" ng-blur="newCategory($event)"></input>
+    <input id="checklist-category" placeholder="Add a Tag" ng-model="template.template_category.name" type="text" class="form-control" typeahead-editable="false" typeahead="category.name for category in categories | filter:$viewValue | limitTo: 5" typeahead-on-select="saveCategory($item)" ng-blur="newCategory($event)"></input>
```

Comment:
The issue mentioned that "Add a Category" is too abstract. Therefore I renamed it to "Tag" instead because it communicates what the form asks for better. This is due to the rise of Twitter's #hashTag tagging (I think). Sound Cloud also follows this nomenclature when a user uploads a sound file. However, they just say "Tags:" as opposed to "Add a Tag," I kept "Add a" to keep a consistent flow on the words.